units/XNL0303/XNL0303_Script.lua

```diff
@@ -1,18 +1,20 @@
--- T3 tank
-
+local NWep = import('/lua/nomadsweapons.lua')
+local NUtils = import('/lua/nomadsutils.lua')
 local NLandUnit = import('/lua/nomadsunits.lua').NLandUnit
-local AnnihilatorCannon1 = import('/lua/nomadsweapons.lua').AnnihilatorCannon1
-local RocketWeapon1 = import('/lua/nomadsweapons.lua').RocketWeapon1
-local AddRapidRepair = import('/lua/nomadsutils.lua').AddRapidRepair
-local AddRapidRepairToWeapon = import('/lua/nomadsutils.lua').AddRapidRepairToWeapon
+
+local AnnihilatorCannon1 = NWep.AnnihilatorCannon1
+local RocketWeapon1 = NWep.RocketWeapon1
+local AddRapidRepair = NUtils.AddRapidRepair
+local AddRapidRepairToWeapon = NUtils.AddRapidRepairToWeapon
 
 NLandUnit = AddRapidRepair(NLandUnit)
 
+--- Tech 3 Tank
```

Comment:
https://github.com/FAForever/nomads/blob/80f08f97381ececa90de9f6e8d5b06946513e7ec/units/XNL0303/XNL0303_unit.bp#L2
```suggestion
--- Tech 3 Hover Tank
```
To differentiate it from XNL0305